GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -578,6 +581,17 @@ protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
             return base.ProcessCmdKey(ref msg, keyData);
         }
 
+        public void LoadCustomDifftools()
+        {
+            List<CustomDiffMergeTool> menus = new ()
+            {
+                new (openWithDifftoolToolStripMenuItem, UnstagedOpenWithDifftoolToolStripMenuItem_Click),
+                new (stagedOpenDifftoolToolStripMenuItem9, StagedOpenDifftoolToolStripMenuItem_Click),
+            };
+
+            CustomDiffMergeTool.LoadCustomDiffMergeTools(Module, menus, components, true, true);
```

Comment:
For GH and those without Re#:
```suggestion
            CustomDiffMergeTool.LoadCustomDiffMergeTools(Module, menus, components, isDiff: true, useSettings: true);
```

Reply:
Thanks!